lib/dalli/protocol/value_serializer.rb

```diff
@@ -33,55 +33,17 @@ def store(value, req_options, bitflags)
         [store_value, bitflags]
       end
 
-      # TODO: Some of these error messages need to be validated.  It's not obvious
-      # that all of them are actually generated by the invoked code
-      # in current systems
-      # rubocop:disable Layout/LineLength
-      TYPE_ERR_REGEXP = %r{needs to have method `_load'|exception class/object expected|instance of IO needed|incompatible marshal file format}.freeze
-      ARGUMENT_ERR_REGEXP = /undefined class|marshal data too short/.freeze
-      NAME_ERR_STR = 'uninitialized constant'
-      # rubocop:enable Layout/LineLength
-
-      def retrieve(value, bitflags) # rubocop:disable Metrics/MethodLength
+      def retrieve(value, bitflags)
         serialized = (bitflags & FLAG_SERIALIZED) != 0
         if serialized
-          if serializer.is_a?(Marshal)
-            begin
-              serializer.load(value)
-            rescue StandardError => e
-              raise UnmarshalError, "Unable to unmarshal value: #{e.message}"
-            end
-          else
-            # Use Dalli's existing exception filtering for deserialization when not using Marshal to serialize.
+          begin
             serializer.load(value)
+          rescue StandardError
```

Comment:
```suggestion
          rescue
```

This is identical (the default is `StandardError`).

Reply:
Yes, _and_, Rubocop rules in this project say

```

Offenses:

lib/dalli/protocol/value_serializer.rb:41:11: C: [Correctable] Style/RescueStandardError: Avoid rescuing without specifying an error class.
          rescue
          ^^^^^^
```

Reply:
Haha never mind then :)